tests/phpunit/tests/term/getTerms.php

```diff
@@ -2589,6 +2609,75 @@ public function test_pad_counts() {
 		}
 	}
 
+	/**
+	 * @ticket 55837
+	 */
+	public function test_pad_counts_cached() {
```

Comment:
The assertions in this test method need a `message` parameter to indicate which assertion failed and went wrong.

Reply:
Done!

Reply:
Done!